app/build.gradle

```diff
@@ -28,6 +28,8 @@ android {
         versionCode 1
         versionName "1.0.0"
 
+        multiDexEnabled true
```

Comment:
What is Multidex for?

Reply:
When the limit of 64k methods is reached on Android, the application can't be compiled as a single classes.dex bytecode, in order to fix the problem, multidex lib+configuration is required. The problem occurs when adding multiples libraries with a large method count as dependency.